subprojects/docs/src/docs/userguide/gradleWrapper.adoc

```diff
@@ -162,23 +256,34 @@ You can download the `.sha256` file by clicking on one of the `sha256` links on
 * https://gradle.org/release-candidate
 * https://gradle.org/nightly
 
++++++
+<figure xmlns:xi="http://www.w3.org/2001/XInclude">
```

Comment:
From previous suggestion that would make this image unnecessary.

Reply:
OK